collect-commit-labels.js

```diff
@@ -19,47 +18,43 @@ function collectCommitLabels (list, callback) {
   }
 
   ghauth(authOptions, (err, authData) => {
-    const cache = {}
-
     if (err) {
       return callback(err)
     }
 
-    const q = async.queue((commit, next) => {
-      function onFetch (err, issue) {
-        if (err) {
-          console.error('Error fetching issue #%s: %s', commit.ghIssue, err.message)
-          return next()
-        }
+    const allPrs = list.map(item => item.ghIssue)
```

Comment:
this is going to cause issues because you are using the full list and not the sublist.

Reply:
(Sublist removes commits without meta data)